integration/nevermined/NFTTemplatesWithEther.test.ts

```diff
@@ -93,8 +93,10 @@ describe('NFTTemplates With Ether E2E', async () => {
         ;({ nftSalesTemplate, nftAccessTemplate } = nevermined.keeper.templates)
 
         // eth
-        nftPrice = Number(web3Utils.toWei(String(nftPrice), 'ether'))
-        amounts = amounts.map(v => new BigNumber(web3Utils.toWei(String(v), 'ether')))
+        nftPrice = ethers.utils.parseEther(nftPrice.toString()).toNumber()
+        amounts = amounts.map(
+            v => new BigNumber(ethers.utils.parseEther(v.toString()).toString())
```

Comment:
weird line, same here, maybe we can isolate this kind of tricky type adaptations in a utility class. That would make easier a further migration to BN.js

Reply:
created new issue for that https://github.com/nevermined-io/sdk-js/issues/308